src/extenders.rs

```diff
@@ -23,7 +23,7 @@ impl<'a> PickleDbListExtender<'a> {
     ///
     /// # Examples
     ///
-    /// ```no_run
+    /// ```ignore
```

Comment:
why change `no_run` to `ignore`?

`no_run` just doesn't run the code, but `ignore` doesn't even build it 🤔